components/schemas/scoped-variables/ScopedVariable.yml

```diff
@@ -26,21 +25,10 @@ properties:
   identifier:
     type: string
     description: An identifier, similar to a key in an environment variable.  Its used when envoking the scoped variable.
-  secret:
-    nullable: true
-    type: object
-    description: An object with information about the encryption of the scoped variable.
-    required:
-      - encrypted
-    properties:
-      encrypted:
-        type: boolean
-        description: A boolean where true means the scoped variables value is encrypted.
-      hint:
-        type: string
-        description: A hint for the decryption password.
   scope:
     "$ref": "./ScopedVariableScope.yml"
+  access:
+    $ref: ./ScopedVariableAccess.yml
   source:
     type: object
     description: The source or value of the scoped variable.
```

Comment:
Source is technically `nullable`:
![image](https://github.com/cycleplatform/api-spec/assets/5110855/f380a55c-d74b-4257-b650-e5c5ad2ae4f2)